app/src/main/java/ch/epfl/qedit/view/edit/EditQuestionActivity.java

```diff
@@ -53,8 +72,29 @@ protected void onCreate(Bundle savedInstanceState) {
         setDoneButtonListener();
         setCancelButtonListener();
 
-        // Get the StringPool, the title and the text from the Intent
+        // Get the StringPool, the title and the text and treasure hunt from the Intent
         extractFromIntent();
+
+        setTreasureHunt();
+    }
+
+    private void setTreasureHunt() {
+        longitudeText = findViewById(R.id.longitude_text);
+        latitudeText = findViewById(R.id.latitude_text);
+        radiusText = findViewById(R.id.radius_text);
+        button_choose_location = findViewById(R.id.edit_choose_location);
+
+        if (isTreasureHunt) {
+            // TODO: Do something... maybe
```

Comment:
Do what?

Reply:
Oups, it was a reminder for me. So that I didn't forget to do something up there